rocky/assets/js/renderNormalizerOutputOOIs.js

```diff
@@ -11,11 +11,14 @@ buttons.forEach((button) => {
     .closest("tr")
     .getAttribute("data-task-id")
     .replace(/-/g, "");
+  const organization =
+    organization_code ||
+    button.closest("tr").getAttribute("data-organization-code");
```

Comment:
The "closest tr" is selected for `raw_task_is` and `task_id` as well, so you could consider reusing it. You could also consider hardening against there not being a `<tr>`. E.g.:

```js
const tr = button.closest("tr");
if (!tr) return;
const raw_task_id = tr.closest("tr").getAttribute("data-task-id");
const task_id = raw_task_id.replace(/-/g, "");
const organization = organization_code || tr.getAttribute("data-organization-code");
```

Reply:
@underdarknl can you comment on this? It does not seem a blocking issue to me, especially since this issue was already present

Reply:
@ppvg true, we will fix this in a new ticket, as @zcrt mentions, it's not a new regression, but they are useful points to improve on.
@TwistMeister could you pick this up?